src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -44,15 +45,16 @@
         "general-commissioning-cluster.xml",
         "general-diagnostics-cluster.xml",
         "global-attributes.xml",
-        "groups-cluster.xml",
         "group-key-mgmt-cluster.xml",
+        "groups-cluster.xml",
         "identify-cluster.xml",
         "illuminance-measurement-cluster.xml",
         "input-output-value-clusters.xml",
         "keypad-input-cluster.xml",
         "level-control-cluster.xml",
         "localization-configuration-cluster.xml",
         "low-power-cluster.xml",
+        "matter-devices.xml",
```

Comment:
Again, this one, not being a cluster XML, should perhaps go at the end.

Reply:
Reverted the sorting part of the PR